src/test/java/com/exasol/adapter/document/DynamodbVsExasolTestDatabaseBuilder.java

```diff
@@ -0,0 +1,67 @@
+package com.exasol.adapter.document;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.sql.SQLException;
+import java.util.concurrent.TimeoutException;
+
+import com.exasol.bucketfs.BucketAccessException;
+
+/**
+ * DynamoDB specific test database builder.
+ */
+public class DynamodbVsExasolTestDatabaseBuilder extends ExasolTestDatabaseBuilder {
+    private static final String VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION = "document-virtual-schema-dist-0.3.0-dynamodb-0.5.0.jar";
+    private static final Path PATH_TO_VIRTUAL_SCHEMAS_JAR = Path.of("target", VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION);
+
+    public DynamodbVsExasolTestDatabaseBuilder(final ExasolTestInterface testInterface)
+            throws SQLException, IOException {
+        super(testInterface);
+    }
+
+    /**
+     * Uploads the dynamodb adapter jar to the exasol test container.
+     */
+    public void uploadDynamodbAdapterJar() throws InterruptedException, BucketAccessException, TimeoutException {
+        this.testInterface.uploadFileToBucketfs(PATH_TO_VIRTUAL_SCHEMAS_JAR, VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION);
+        uploadJacocoAgent();
+    }
+
+    public void createUdf() throws SQLException {// TODO make DynamoDB independent
+        final StringBuilder statementBuilder = new StringBuilder("CREATE OR REPLACE JAVA SET SCRIPT ")
+                .append(ADAPTER_SCHEMA).append(".").append(UdfRequestDispatcher.UDF_PREFIX + "DYNAMO_DB").append("(")
+                .append(AbstractDataLoaderUdf.PARAMETER_DOCUMENT_FETCHER).append(" VARCHAR(2000000), ")
+                .append(AbstractDataLoaderUdf.PARAMETER_REMOTE_TABLE_QUERY).append(" VARCHAR(2000000), ")
+                .append(AbstractDataLoaderUdf.PARAMETER_CONNECTION_NAME).append(" VARCHAR(500)) EMITS(...) AS\n");
+        addDebuggerOptions(statementBuilder, true);
+        statementBuilder.append("    %scriptclass ").append(UdfRequestDispatcher.class.getName()).append(";\n");
+        statementBuilder.append("    %jar /buckets/bfsdefault/default/").append(VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION)
+                .append(";\n");
+        statementBuilder.append("/");
+        final String sql = statementBuilder.toString();
```

Comment:
It's very hard to read this builder concatenations. Maybe it's better to use just a string instead?

Reply:
What exactly would you suggest?
I learned that += leads to bad performance...
Using + inside of an append leads to sonar warnings which suggest using the builder instead.

Reply:
It a test, so such a small performance optimization doesn't really matter here  and sometimes you can ignore this warning if it significantly improves readability